hyppo/discrim/discrim_one_samp.py

```diff
@@ -56,7 +56,7 @@ def statistic(self, x, y):
 
         Parameters
         ----------
-        x,y : ndarray
+        x,y : ndarray of floats
```

Comment:
change from `floats` to `float` everywhere, since writing `float` actually links to the Python documentation.

Reply:
ok Sure.